sphinx/ext/inheritance_diagram.py

```diff
@@ -408,4 +615,6 @@ def setup(app):
     app.add_config_value('inheritance_graph_attrs', {}, False),
     app.add_config_value('inheritance_node_attrs', {}, False),
     app.add_config_value('inheritance_edge_attrs', {}, False),
+    app.add_config_value('inheritance_node_attrs_builtin', {}, False),
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expression "(app.add_config_value('inheritance_node_attrs_builtin', {}, False), )" is assigned to nothing](https://www.codacy.com/app/georg/sphinx/file/2138166258/issues/source?bid=3149936&fileBranchId=3149963#l618)